testdata/p4_16_samples_outputs/issue363-bmv2-first.p4

```diff
@@ -0,0 +1,68 @@
+#include <core.p4>
```

Comment:
This file does not correspond to any test. Please delete the spurious output files.

Reply:
Done.

Reply:
This think you are asking for 'include core.p4' to be removed from the P4 program.

I see that 'make check' seems to be adding '#include <core.p4' to the p4c/testdata/p4_16_sample_outputs/issue383-bmv2-first.p4 because in p4c/testdata/p4_16_samples/issue383-bmv2-first.p4, we have '#include <v1model.p4>' which includes core.p4.  

Or, sorry, I don't understand the comment.

Reply:
There is no test called issue363-bmv2.p4. These are output files that should not be part of this commit, they are probably left-overs when you had a file mistakenly named like that. The issue is 383.

Reply:
ok, got it, thanks! I will remove these files in next commit.